src/api/app/services/scm_status_reporter.rb

```diff
@@ -0,0 +1,49 @@
+class SCMStatusReporter
+  attr_accessor :payload, :scm_token, :state
+
+  def initialize(payload, scm_token, event = nil)
```

Comment:
*nitpick* `event=nil` to convey that this is the initial reporting is kind of hard to understand. Also makes setting `@state` below hard to understand.